hedyorg/hedy
# [UI] Allows ordering of adventures per level

**Description**

Introduces a more granular way to order adventures in each one of the levels. This is still hard coded in `hedy_content.py`, in the future we want this to be retrieved from the database so every teacher can customize this.

For now, to change the ordering of the adventures you can modify them in `ADVENTURE_ORDER_PER_LEVEL`
If someone adds a new adventure in a level, they should add the adventure to this dictionary too.

**Fixes #3535**

**How to test**

I left the ordering the same as it already is, to change it and see if the test really works you can change the position of one of the adventures in one of the levels and then see if it's changed in the site.

**Checklist**
Done? Check if you have it all in place using this list:*
  

If you're unsure about any of these, don't hesitate to ask. We're here to help!
